Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -445,6 +445,7 @@ extension PlaygroundController {
             "use_link": settings.linkEnabled == .on,
             "use_manual_confirmation": settings.integrationType == .deferred_mc,
             "require_cvc_recollection": settings.requireCVCRecollection == .on,
+            // "supported_payment_methods": ["card", "link"], // Uncomment to override payment methods (also make sure Automatic PM's is off)
```

Comment:
for now, this needs to be uncommented for Instant Debits to show up via playground

there's a precedent for it below (😁 ):

> // "set_shipping_address": true // Uncomment to make server vend PI with shipping address populated

likely not the end state we want but its OK for now given the state of the project (still iterating on design etc.)

Reply:
Does turning the Link flag on work? That should set `[card, link]` as the PMs. If this is necessary, might be nice to make it a switch, it's not too difficult to add a new one

Reply:
@davidme-stripe 

What is the Link flag? Or, in other words, given the current state, what would I change/do to try turning the Link flag on? (also asked some other folks in the meanwhile too)

I do see this (...but doesn't do anything haha):

![Screenshot 2024-05-17 at 11 25 13 AM](https://github.com/stripe/stripe-ios/assets/105514761/f6103303-7967-4128-9ccb-1c18854fb874)

Reply:
That should be it — if it's `on`, the backend should send `[card, link, (a few others)]`.

Reply:
@davidme-stripe 

With that on...

in:

```
static func filteredPaymentMethodTypes(from intent: Intent, configuration: Configuration, logAvailability: Bool = false) -> [PaymentMethodType]
```

`intent.recommendedPaymentMethodTypes` returns:

```
(lldb) po intent.recommendedPaymentMethodTypes
▿ 7 elements
  - 0 : card
  - 1 : link
  - 2 : afterpayClearpay
  - 3 : USBankAccount
  - 4 : klarna
  - 5 : Amazon Pay
  - 6 : affirm
```

they key problem, as I understand it, is that the "USBankAccount" is returned (where if that exists, Instant Debits is disabled)

LMK if there's other easy way around this, right now Android added a comma-separated list to the playground app to support this through the UI and the current plan was to just [copy it](https://github.com/stripe/stripe-ios/pull/3585): https://github.com/stripe/stripe-android/pull/8428

Reply:
Oh, got it! That makes sense to me.